pyprep/ransac.py

```diff
@@ -122,6 +122,9 @@ def find_bad_by_ransac(
         err = "Argument 'n_samples' must be an int (got {0})"
         raise TypeError(err.format(type(n_samples).__name__))
 
+    if isinstance(complete_chn_labs, list):
+        complete_chn_labs = np.array(complete_chn_labs)
```

Comment:
this is a bit cleaner (see: https://numpy.org/doc/stable/reference/generated/numpy.asarray.html)

```suggestion
    complete_chn_labs = np.asarray(complete_chn_labs)
```